test/blackbox/common/BlackboxTestsPubSubBasic.cpp

```diff
@@ -81,6 +86,51 @@ class PubSubBasic : public testing::TestWithParam<std::tuple<communication_type,
 
 };
 
+/**
+ * @brief Test function for EnvFileWarning* tests
+ *
+ * Sets environment variable @c FASTDDS_ENVIRONMENT_FILE to @c env_file_name, and then initializes a DomainParticipant,
+ * using a custom log consumer to check how many logWarnings matching
+ * ".*does not exist. File watching not initialized." are issued, comparing that number with @c expected_logs.
+ * Finally, it resets the log module
+ *
+ * @param env_file_name The value to assign to @c FASTDDS_ENVIRONMENT_FILE
+ * @param expected_logs The number of expected logs (error or warning) matching
+ *                      ".*does not exist. File watching not initialized."
+ */
+void env_file_warning(const char* env_file_name, size_t expected_logs)
```

Comment:
Linters complain with this.

```suggestion
void env_file_warning(
        const char* env_file_name,
        size_t expected_logs)
```